### PR TITLE
some css fixes + cleanup

### PR DESCRIPTION
**Pull request by nicolasbettag**
i fixed some css stuff - there is also some css cleanup.

### CODE REVIEW

**Comment by SickRage2** [comment]
Thanks for the pull request! Before a real human comes by, please make sure your PR has all of the below criteria checked
- [x] Give a description on what the PR is for.
- [x] Make sure your PR is based on the DEVELOP branch
- [x] Don't send big changes all at once. Split up big PRs into multiple smaller PRs that are easier to manage and review

Please make sure you also read [contribution guide](https://github.com/SickRage/SickRage/blob/master/contributing.md) and followed all of the steps.

Thanks!

**Comment by neoatomic** [comment]
Thanks for the help @nicolasbettag. :)

**Comment by labrys** [comment]
@nicolasbettag My css is rusty since I haven't used it in quite a while, but what is the benefit in removing the browser specific compatibility code?

File: gui/slick/css/dark.css [hunk 1]
```diff
@@ -283,9 +283,6 @@ home.mako
     top: 0;
     right: 0;
     background-image: -webkit-linear-gradient(right, rgba(51, 51, 51, 1), rgba(51, 51, 51, 0));
-    background-image: -moz-linear-gradient(right, rgba(51, 51, 51, 1), rgba(51, 51, 51, 0));
```
Comment (OmgImAlexis) [inline gui/slick/css/dark.css:286]: Unless you can link me to something that shows none of these are needed this needs to be added back.

**Comment by nicolasbettag** [comment]
Well, I'm usually working with https://github.com/iamvdo/pleeease. The most browsers are able to support transitions, etc. without prefixing.

File: gui/slick/css/dark.css (hunk 1 above)
Comment (nicolasbettag) [inline gui/slick/css/dark.css:286]: @labrys the most browsers are now able to support it without prefixes. so the load time is faster and there aren't fallbacks, etc.

Comment (labrys) [inline gui/slick/css/dark.css:286]: @nicolasbettag the key word there is _most_.  The reason for the fallback is to try to support all, especially considering the wide range of devices we support.  I'm all for removing unnecessary code and improving performance while simplifying code.  My concern is if this breaks compatibility for a platform.

Comment (nicolasbettag) [inline gui/slick/css/dark.css:286]: @labrys Keep my PR in mind. I'm using https://github.com/iamvdo/pleeease.

**Comment by OmgImAlexis** [comment]
So I'm guessing this was done with the tool instead of by hand? I think for now we'll just leave it as is. The plan is to wait till Bootstrap 4 is out and then redo everything with SASS from the ground-up anyways.

**Pull request closed without merging**